content/chef_compliance_phase.md

```diff
@@ -13,33 +13,33 @@ draft = false
 
 [\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/main/content/chef_compliance_phase.md)
 
-Chef Infra Client's Compliance Phase lets you automatically execute compliance audits and view the results as part of any Chef Infra Client run. The Compliance Phase of the Chef Infra Client run replaces the legacy [audit cookbook](https://supermarket.chef.io/cookbooks/audit) and works with your existing audit cookbook attributes, and you can also set it up for new cookbooks. This additional phase gives you the latest compliance capabilities without having to manage cookbook dependencies or juggle versions during Chef Infra Client updates.
+Chef Infra Client's compliance phase lets you automatically execute compliance audits and view the results as part of any Chef Infra Client run. The compliance phase of the Chef Infra Client run replaces the legacy [audit cookbook](https://supermarket.chef.io/cookbooks/audit) and works with your existing audit cookbook attributes, and you can also set it up for new cookbooks. This additional phase gives you the latest compliance capabilities without having to manage cookbook dependencies or juggle versions during Chef Infra Client updates.
```

Comment:
We've been using Compliance Phase as a marketed feature name that's been capitalized as we talk about it externally similar to Policyfiles

Reply:
Not my favorite choice, but I will make it so.